libs/market-list/src/lib/components/select-market-table.tsx

```diff
@@ -31,26 +32,44 @@ export const SelectMarketTableHeader = ({
 export const SelectMarketTableRow = ({
   detailed = false,
   columns,
+  onSelect,
+  marketId,
 }: {
   detailed?: boolean;
   columns: Column[];
+  onSelect: (id: string) => void;
+  marketId: string;
 }) => {
-  return (
-    <tr
-      className={`hover:bg-neutral-200 dark:hover:bg-neutral-700 cursor-pointer relative h-[34px]`}
-    >
-      {columns.map(({ kind, value, className, dataTestId, onlyOnDetailed }) => {
-        if (!onlyOnDetailed || detailed === onlyOnDetailed) {
-          const tdClass = classNames(className);
-          return (
-            <td key={kind} data-testid={dataTestId} className={tdClass}>
-              {value}
-            </td>
-          );
-        }
+  const handleKeyPress = (event: React.KeyboardEvent, id: string) => {
+    if (event.key === 'Enter' && onSelect) {
+      return onSelect(id);
+    }
+  };
 
-        return null;
-      })}
-    </tr>
+  return (
+    <Link href={`/markets/${marketId}`} passHref={true}>
```

Comment:
chore (blocking): Two things here.

1. Pretty sure this results in invalid html <a> cannot wrap a <tr>.
2. The imported link tag is the nextjs link tag. This is probably fine for now, but if other apps use it might cause issues. I think the easiest thing is to just call call `router.push('/market/id')` in the onSelect handler in the nextjs page. This way the responsiblity of routing is left to the app only